### PR TITLE
Device Limit added

### PR DESCRIPTION
**Pull request by swapnilsparsh**
- Device Limit added

### CODE REVIEW

File: daemon/protocol/protocol.go [hunk 1]
```diff
@@ -952,6 +959,52 @@ func (p *Protocol) processRequest(conn net.Conn, message string) {
...
+			resp = types.DeviceListResponse{
+				APIStatus: apiCode,
+			}
+		} else {
+			// Success. Sending session info
+			resp = types.DeviceListResponse{
+				APIStatus:       apiCode,
+				APIErrorMessage: apiErrMsg,
+				Account:         accountInfo,
+				Data:            data.Data,
```
Comment (swapnilsparsh) [inline daemon/protocol/protocol.go:998]: Response Error

**Comment by kocmo** [comment]
This branch doesn't compile for me, fixing

**Comment by kocmo** [comment]
Merging into master. This stack still remains a stub for now.

**Review by kocmo — APPROVED** [review]
Merging into master. This stack still remains a stub for now.

**Pull request merged**